docs/extend/authorization.md

```diff
@@ -45,6 +45,9 @@ Authorization plugins must follow the rules described in [Docker Plugin API](plu
 Each plugin must reside within directories described under the 
 [Plugin discovery](plugin_api.md#plugin-discovery) section.
 
+Note: the abbreviations `AuthZ` and `AuthN` mean authorization and authentication
```

Comment:
Yes we do mention `AuthN` in the API `UserAuthNMethod` in `docs/extend/authorization.md`